ert_shared/storage/extraction.py

```diff
@@ -14,25 +14,22 @@
 logger = logging.getLogger()
 
 
-def create_experiment(ert) -> Mapping[str, Any]:
-    return dict(name=str(datetime.datetime.now()))
+def create_experiment(ert) -> dict:
```

Comment:
Why the reduction of specificity? Can't we say anything about what `Any` is?

Reply:
like Mapping[str, Union[str, List[dict]]]? At some point it just becomes confusing again...

Reply:
On a general basis I would say that is a good thing, because the confusion is honest. It is a consequence of one dragging the basic types in Python way too far... But, it is in line with the rest of the typing in this file so I'll leave it for now :)